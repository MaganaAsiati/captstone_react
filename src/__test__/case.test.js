import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { HashRouter as Router } from 'react-router-dom';
import store from '../Redux/configureStore';
import Home from '../components/cases/case';

describe('home Component test', () => {
  it('Renders Mission Row', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Home />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
