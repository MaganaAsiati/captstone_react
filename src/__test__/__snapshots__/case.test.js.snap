// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home Component test Renders Mission Row 1`] = `
<div
  className="all-countries"
>
  <div>
    <div
      className="searbar"
    >
      <form>
        <input
          className="input-search"
          onChange={[Function]}
          placeholder="search a country..."
          type="search"
          value=""
        />
      </form>
    </div>
  </div>
  <div
    className="country-container"
  />
</div>
`;
